velox/functions/sparksql/tests/ComparisonsTest.cpp

```diff
@@ -73,10 +73,61 @@ class ComparisonsTest : public SparkFunctionBaseTest {
     auto actual = evaluate<SimpleVector<bool>>(
         fmt::format("{}(c0, c1)", functionName), makeRowVector(input));
     facebook::velox::test::assertEqualVectors(expectedResult, actual);
-  };
+  }
+
+  RowVectorPtr constructRowVector(
+      const std::optional<int64_t>& value,
+      const bool rowIsNull = false) {
```

Comment:
rowIsNull ->rowHasNull

Reply:
We want to test the case that row is null, so here we add this parameter to pass to `makeRowVector` to tell if rows are nulls. I think this does not indicate that row contains nested null.

Reply:
Can we check if value is std::nullopt to determine if row is null?

Reply:
For example `row(bigint)`, we need to test both cases: row(null) and null. So I need extra parameter to tell if row self is null.